include/IECore/SplineParameter.h

```diff
@@ -41,6 +41,12 @@
 namespace IECore
 {
 
+// \todo : Using Spline*Data has basically been deprecated in favour of Ramp*Data.
+// Setting up data this way will no longer work with renderer backends. In theory,
+// the right thing to do might be to replace this with RampParameter ... but on
+// the other hand, Parameter itself probably should be deprecated ( these days,
+// anyone using Cortex to build UIs is probably using Gaffer ).
+
```

Comment:
I agree that we shouldn't be adding RampParameter, and that the future is the removal of Parameters entirely. In which case I don't think there's really a todo here...